tests/reader.rs

```diff
@@ -0,0 +1,75 @@
+use osmpbfreader;
+use std::fs::File;
+
+const PBF: &str = "tests/clipperton.osm.pbf";
+
+#[test]
+fn read_pbf() {
+    let file = File::open(PBF).unwrap();
+    let mut pbf = osmpbfreader::OsmPbfReader::new(file);
+
+    let mut num_nodes = 0;
+    let mut num_ways = 0;
+    let mut num_relations = 0;
+
+    for obj in pbf.iter() {
+        let obj = obj.unwrap();
+        match obj {
+            osmpbfreader::OsmObj::Node(_) => num_nodes += 1,
+            osmpbfreader::OsmObj::Way(_) => num_ways += 1,
+            osmpbfreader::OsmObj::Relation(_) => num_relations += 1,
+        }
+    }
+    assert_eq!(num_nodes, 3097);
+    assert_eq!(num_ways, 151);
+    assert_eq!(num_relations, 17);
+}
+
+#[test]
+fn read_pbf_par_iter() {
+    let file = File::open(PBF).unwrap();
+    let mut pbf = osmpbfreader::OsmPbfReader::new(file);
+
+    let mut num_nodes = 0;
+    let mut num_ways = 0;
+    let mut num_relations = 0;
```

Comment:
I think you can do a `struct Stats` with these fields (and maybe also sum_ids) and implement `FromIterator<OsmObj> for Stats`

Reply:
Good idea, as it would allow to share the expected values between tests.

Reply:
You didn’t do that.